### DISCUSSION TITLE
Tracing not working when set internalMiddlewares:false

### DISCUSSION THREAD

**Discussion by hienbase** [moleculerjs/moleculer#1128] [Q&A] [answered]
When i set param internalMiddlewares: false (moleculer.config.js). Tracing console not working

**Comment by intech** [comment, marked as answer]
@hienbase tracing is one of the [built-in middleware](https://moleculer.services/docs/0.14/middlewares.html#Internal-middlewares).

**Reply by hienbase** [reply]
![image](https://user-images.githubusercontent.com/111110592/186741758-680d1862-67bd-4a41-84b2-a63b85d4349c.png)
i had add it in middlewares[]

**Reply by intech** [reply]
Add this
https://github.com/moleculerjs/moleculer/blob/02d6072b822ae7829e839b92e89a24048b3597db/src/middlewares/index.js#L21

**Reply by hienbase** [reply]
@intech Thank you, it worked!

**Reply by intech** [reply]
@hienbase Thanks for pointing out the missed in the documentation. [I've updated it!](https://github.com/moleculerjs/site/commit/af275f13b191fed4cbfd3d748d8fba601a1ffb35)
